.gitignore-revs

```diff
@@ -1,3 +1,4 @@
 c76131748cbb048e78a987eb234d8a9609e26430
 f38d8cf7cac992767a451d2f3cf3310e3546bf6f
 fa80f12bf3fbfe1337ae69e12c59051fac347001
+99e64a8e990d2579bcfe382f755edbc61f70af4e
```

Comment:
Was this on purpose?

Reply:
Yes...the types document still had our merge commit from version 10.5.0

Reply:
Somehow that seems to exist across multiple SHAs.